rules/autoconsent/complianz-optin.json

```
@@ -12,7 +12,8 @@
         { "click": ".cc-settings" },
         { "waitForVisible": "[aria-label=\"cookies preferences popup\"]" },
         { "click": "[aria-label=\"cookies preferences popup\"] input[type=checkbox]:not([disabled])", "all": true },
-        { "click": "[aria-label=\"cookies preferences popup\"] [aria-label=\"Accept Selected\"]" }
+        { "click": "[aria-label=\"cookies preferences popup\"] [aria-label=\"Accept Selected\"]", "optional": true },
+        { "click": "[aria-label=\"cookies preferences popup\"] [aria-label=\"Save my choice\"]", "optional": true } 
```

```suggestion
        { "click": "[aria-label=\"cookies preferences popup\"] [aria-label=\"Accept Selected\"], [aria-label=\"cookies preferences popup\"] [aria-label=\"Save my choice\"], .cc-btn-accept-selected, .cc-deny", "optional": true }
```
We can combine these lines because we only want to click one of these.

---

Change made and working